controllers/ldap.go

```diff
@@ -238,7 +238,9 @@ func (c *ApiController) UpdateLdap() {
 
 	var ldap object.Ldap
 	err := json.Unmarshal(c.Ctx.Input.RequestBody, &ldap)
-	if err != nil || util.IsStringsEmpty(ldap.Owner, ldap.ServerName, ldap.Host, ldap.Username, ldap.Password, ldap.BaseDn) {
+	if err != nil || util.IsStringsEmpty(ldap.Owner, ldap.ServerName, ldap.Host, ldap.BaseDn) ||
```

Comment:
If put complicated condition to meanful varables it will be easy to understand common sence 

```
anyRequiredLdapFieldEmpty := util.IsStringsEmpty(ldap.Owner, ldap.ServerName, ldap.Host, ldap.BaseDn)
disabledCryptoAndEmptyCred := !ldap.EnableCryptographicAuth && util.IsStringsEmpty(ldap.Username, ldap.Password)
enabledCryptoAndEmptyCert := ldap.EnableCryptographicAuth && len(ldap.ClientCert) == 0

if err != nil || anyRequiredLdapFieldsAreEmpty || disabledCryptoAndEmptyCred || enabledCryptoAndEmptyCert {
		msg := c.T("general:Missing parameter")
```